packages/main/src/components/MessageBox/index.tsx

```diff
@@ -235,7 +231,7 @@ const MessageBox = forwardRef((props: MessageBoxPropTypes, ref: Ref<DialogDomRef
       slot={slot}
       ref={ref}
       style={style}
-      title={tooltip ?? props.title}
+      title={props.title}
```

Comment:
We don't need to set `title` here anymore, please remove it here and in the destructed assignment (`  const { footer, headerText, title, onAfterClose, ...restWithoutOmitted } = rest;`)